topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,429 @@
+3. **paternal_id**: The sample name of the father or 0 for missing.
+4. **maternal_id**: The sample name of the mother or 0 for missing.
+5. **sex**: The sex of the person.
+6. **phenotype**: Wether or not the person is affected by the disease.
+
+> ### {% icon hands_on %} Hands-on: Creating the PED file
+>
+> 1. Upload the pedigree file from below.
+> ```
+> #family_id	name	paternal_id	maternal_id	sex	phenotype
+> FAM0001822	Case6M	0	0	2	2
+> FAM0001822	Case6F	0	0	1	1
+> FAM0001822	Case6C	Case6F	Case6M	2	2
+> ```
+> 
+> {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+{: .hands_on}
+
+For more information on the PED file you can read the help section of the {% tool [GEMINI load](toolshed.g2.bx.psu.edu/repos/iuc/gemini_load/gemini_load/0.20.1+galaxy2) %} tool in the description, which can be found at the bottom of the page when clicking on the tool.
+
+## Load GEMINI database
+Now we can transform the subsampled VCF and PED file into a GEMINI database. Note that this can take a very long time depending on the size of the VCF. In our case it should take around 30-40 minutes.
+
+> ### {% icon hands_on %} Hands-on: Transform VCF and PED files into a GEMINI database
+>
+> 1. {% tool [GEMINI load](toolshed.g2.bx.psu.edu/repos/iuc/gemini_load/gemini_load/0.20.1+galaxy2) %} with the following parameters:
+>    - {% icon param-file %} *"VCF dataset to be loaded in the GEMINI database"*: `SNPeff annotated VCF` (output of **SnpEff eff** {% icon tool %})
+>    - {% icon param-file %} *"Sample and family information in PED format"*: the pedigree file prepared above
+>
+>
+{: .hands_on}
+
+
+## Find inheritance pattern
+With the GEMINI database it is now possible to identify the causative variant that could explain the breast cancer in the mother and daughter. The inheritance information makes it a bit easier to determine which tool to run to find the causative variant, instead of finding it by trying all the different inheritance patterns. 
+
+> ### {% icon question %} Question
+>
+> Which inheritance pattern could have occurred in this family trio?
+>
+> > ### {% icon tip %} Tip: Possible patterns
+> > The available inheritance patterns can be found in the {% tool [GEMINI inheritance pattern](toolshed.g2.bx.psu.edu/repos/iuc/gemini_inheritance/gemini_inheritance/0.20.1) %} tool.
+> {: .tip}
+>
+> > ### {% icon solution %} Solution
+> >
+> >    - Since both the daughter and mother are affected and the father unaffected and it also being very unlikely that a men has breast cancer, the mutation is most probably dominant. The disease could still be recessive if the father has one copy of the faulty gene, however this is less likely. 

```

Comment:
```suggestion
> >    - Since both the daughter and mother are affected, the father likely unaffected[^1], the mutation is most probably dominant. The disease could still be recessive if the father has one copy of the faulty gene, however this is less likely. 
```

this is the syntax for a footnote, you can later (not inside the `>`s) write:

```
[^1]:  It is unlikely that the father has breast cancer, "For men, the lifetime risk of getting breast cancer is about 1 in 833" [via cancer.org](https://www.cancer.org/cancer/breast-cancer-in-men/about/key-statistics.html)
```